test/spec/import/ImporterSpec.js

```diff
@@ -89,11 +93,14 @@ describe('import - Importer', function() {
       });
 
       // when
-      runImport(diagram, xml, function(err, warnings) {
+      runImport(diagram, xml).then(function() {
```

Comment:
Simply return `runImport(...).then(...)` instead. Mocha handles the rest. Get rid of `done()`.